specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -0,0 +1,2200 @@
+      "enum": [
+        "MANAGED",
+        "EXTERNAL"
+      ],
+      "x-ms-enum": {
+        "name": "TableType",
+        "modelAsString": false
+      }
+    },
+    "SortOrder": {
+      "description": "Sorting order.",
+      "type": "string",
+      "enum": [
+        "DESC",
+        "ASC"
+      ],
+      "x-ms-enum": {
+        "name": "SortOrder",
+        "modelAsString": false
+      }
+    },
+    "RelationshipType": {
+      "description": "Relation Type",
+      "type": "string",
+      "enum": [
+        "ONETOONE",
+        "ONETOMANY",
+        "MANYTOONE",
+        "MANYTOMANY"
+      ],
+      "x-ms-enum": {
+        "name": "RelationshipType",
+        "modelAsString": false
+      }
+    },
+    "ValidationType": {
+      "description": "Validation Type",
+      "type": "string",
+      "enum": [
+        "IDWValidation"
+      ],
+      "x-ms-enum": {
+        "name": "ValidationType",
+        "modelAsString": false
+      }
+    },
+    "ValidationStatus": {
+      "description": "Validation Status",
+      "type": "string",
+      "enum": [
+        "VALID",
+        "INVALID"
+      ],
+      "x-ms-enum": {
+        "name": "ValidationStatus",
+        "modelAsString": false
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "description": "The object that defines the structure of an Azure Synapse error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "x-ms-client-flatten": true,
+          "description": "Error data",
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "description": "The object that defines the structure of an Azure Synapse error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "Property name/path in request associated with error.",
+          "type": "string"
+        },
+        "details": {
+          "description": "Array with additional error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudError"
+          }
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "description": "The Synapse client API Version."
+    },
+    "UserAgentParameter": {
+      "name": "User-Agent",
+      "type": "string",
+      "in": "header",
+      "required": true,
+      "x-ms-parameter-location": "client",
+      "description": "User-Agent request header for servers to identify application"
+    },
```

Comment:
Does synapse have specific requirement on user-agent provided by customer? What if user call REST without user-agent or just CURL user-agent.

Reply:
No specific requirements, CURL requests without User-Agent will fail. 
The CURL request will need to have a user-agent specified with -A for it to go thorugh successfully.